src/pages/wishlist/Wishlist.jsx

```diff
@@ -11,11 +15,31 @@ export function Wishlist() {
           className={`text__center font__regular ${wishlist['wishlist__header']}`}>
           My Wishlist
         </h3>
+        {
+          <div className="text__lg font__bold text__center">
+            {wishlistItems.length === 0 ? (
+              <div className={wishlist['flex__column']}>
+                <div>Your Wishlist is Empty</div>
+                <div>Add Products to Your Wishlist</div>
+                <Link to="/products" className="text__xl">
+                  <button className="btn btn__primary text__lg">
+                    Shop Now
+                  </button>
+                </Link>
+              </div>
+            ) : (
+              <p
+                className={`text__lg font__bold text__center ${wishlist['items__count']}`}>
+                No of items in wish List {wishlistItems.length}
+              </p>
+            )}
+          </div>
+        }
+
         <div className={wishlist['wishlist__container']}>
-          <WishlistCard />
-          <WishlistCard />
-          <WishlistCard />
-          <WishlistCard />
+          {wishlistItems.map(product => (
+            <WishlistCard key={product['id']} {...product} />
```

Comment:
any specific reason for not using dot operator for specifying object values ?

Reply:
No, but now using dot operator